packages/react-docs/gatsby-node.js

```diff
@@ -31,10 +31,7 @@ exports.modifyWebpackConfig = ({ config, stage }) => {
     resolve: {
       alias: {
         '@patternfly/react-core': path.resolve(__dirname, '../react-core/src'),
-        '@patternfly/react-icons': path.resolve(
-          __dirname,
-          '../react-icons/src'
-        ),
+        '@patternfly/react-icons': path.resolve(__dirname, '../react-icons'),
```

Comment:
this can actually be removed since lerna does that for you. I had it before to remove the need to `/src/`, but honestly it is better the way you have it in `icons.js`.  Less indirection.